### DISCUSSION TITLE
View all my pending (started but not submitted) PR reviews

### DISCUSSION THREAD

**Discussion by alete89** [community/community#38359] [Pull Requests]
I, as a teacher, use Github Classroom to give assignments to my students. I use the Feedback PR feature to give them feedback and grades. If the assignment is large enough, it takes several minutes to review, and I might need to pause. Also, in order to not spoil grades progressively so every student get their review at the same time, I leave all my reviews as pending (I don't submit them right away) and then I submit all of them at the same time.

It would be awesome if I could quickly access all the PR reviews that are still pending, to a) take from where I left if I had to leave, and b) make sure I'm not missing anyone :sweat_smile:

**Comment by ossanna16** [comment]
Hi there @alete89 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats
